### PR TITLE
MBS-11041: Make ModBot leave a note on autoremoval edits

### PR DESCRIPTION
**Pull request by reosarevok**
### Implement MBS-11041

These edits often confuse users, who don't understand why their added entity has been removed without a word by some weird editor. This adds an explanatory edit note which should hopefully avoid those situations and help them understand they can readd the entity if needed.

Screenshot from 2020-08-18 15-25-18

@mwiencek: please let me know if localized_note would not work here for some reason :)

### CODE REVIEW

File: lib/MusicBrainz/Script/RemoveEmpty.pm
Comment (mwiencek) [inline lib/MusicBrainz/Script/RemoveEmpty.pm:131]: Oops, meant to request changes. You also have to wrap the string with `N_l` here

File: lib/MusicBrainz/Script/RemoveEmpty.pm
Comment (yvanzo) [inline lib/MusicBrainz/Script/RemoveEmpty.pm:132]: please moderate lines’ length in code

Comment (yvanzo) [inline lib/MusicBrainz/Script/RemoveEmpty.pm:132]: please moderate lines’ length in code

**Review by yvanzo — APPROVED** [review]
LGTM but coding style, it probably doesn’t require a second review though.

Edit: And thanks for the screenshot!

**Pull request merged**
